plugins/webp-uploads/hooks.php

```diff
@@ -467,12 +508,17 @@ function webp_uploads_remove_sources_files( int $attachment_id ): void {
 				continue;
 			}
 
-			if ( ! is_array( $backup_properties ) || empty( $backup_properties['file'] ) ) {
+			if (
+				! is_array( $backup_properties ) ||
+				! isset( $backup_properties['file'] ) ||
```

Comment:
I don't think the first check is needed here? Since the `isset()` check after already satisfies the first.
```suggestion
			if (
				! isset( $backup_properties['file'] ) ||
```